golem-worker-service-base/src/service/worker/default.rs

```diff
@@ -1053,6 +1210,18 @@ where
         })
     }
 
+    async fn get_metadatas(
```

Comment:
Minor. Name is weird. `metadata` is already plural, isn't it :D

Reply:
`get_metadata` / `get_all_metadata`